### PR TITLE
Notifications theme v2

### LINKED ISSUE #98: Notification components
- Info (for angular only)
- Error
- Success

# SDK support
- https://github.com/GetStream/stream-chat-angular/issues/277
- https://github.com/GetStream/stream-chat-react/issues/1587

### PR DESCRIPTION
**Pull request by szuperaz**
### 🎯 Goal

_Describe why we are making this change_

### 🛠 Implementation details

_Provide a description of the implementation_

### 🎨 UI Changes

https://github.com/GetStream/stream-chat-angular/pull/314

closes #98

### CODE REVIEW

File: docs/theming/component-variables.mdx [hunk 1]
```diff
@@ -534,6 +534,54 @@ Defined in: [https://github.com/GetStream/stream-chat-css/blob/main/src/v2/style
...
+| Name                                                | Description                                             |
+| --------------------------------------------------- | ------------------------------------------------------- |
+| `--str-chat__notification-list-font-family`         | The font used in the component                          |
+| `--str-chat__notification-list-color`               | The text/icon color of the component                    |
+| `--str-chat__notification-list-background-color`    | The background color of the component                   |
+| `--str-chat__notification-list-border-radius`       | The border radius used for the borders of the component |
+| `--str-chat__notification-list-border-block-start`  | Top border of the component                             |
+| `--str-chat__notification-list-border-block-end`    | Bottom border of the component                          |
+| `--str-chat__notification-list-border-inline-start` | Left (right in RTL layout) border of the component      |
+| `--str-chat__notification-list-border-inline-start` | Right (lrft in RTL layout) border of the component      |
```
Comment (oliverlaz) [inline docs/theming/component-variables.mdx:548]:
```suggestion
| `--str-chat__notification-list-border-inline-end` | Right (left in RTL layout) border of the component      |
```
Typo

File: docs/theming/component-variables.mdx [hunk 2]
```diff
@@ -534,6 +534,54 @@ Defined in: [https://github.com/GetStream/stream-chat-css/blob/main/src/v2/style
...
+| Name                                           | Description                                             |
+| ---------------------------------------------- | ------------------------------------------------------- |
+| `--str-chat__notification-font-family`         | The font used in the component                          |
+| `--str-chat__notification-color`               | The text/icon color of the component                    |
+| `--str-chat__notification-background-color`    | The background color of the component                   |
+| `--str-chat__notification-border-radius`       | The border radius used for the borders of the component |
+| `--str-chat__notification-border-block-start`  | Top border of the component                             |
+| `--str-chat__notification-border-block-end`    | Bottom border of the component                          |
+| `--str-chat__notification-border-inline-start` | Left (right in RTL layout) border of the component      |
+| `--str-chat__notification-border-inline-start` | Right (lrft in RTL layout) border of the component      |
```
Comment (oliverlaz) [inline docs/theming/component-variables.mdx:564]:
```suggestion
| `--str-chat__notification-border-inline-end` | Right (left in RTL layout) border of the component      |
```

File: docs/theming/component-variables.mdx [hunk 3]
```diff
@@ -534,6 +534,54 @@ Defined in: [https://github.com/GetStream/stream-chat-css/blob/main/src/v2/style
...
+| Name                                      | Description                                             |
+| ----------------------------------------- | ------------------------------------------------------- |
+| `--str-chat__tooltip-font-family`         | The font used in the component                          |
+| `--str-chat__tooltip-color`               | The text/icon color of the component                    |
+| `--str-chat__tooltip-background-color`    | The background color of the component                   |
+| `--str-chat__tooltip-border-radius`       | The border radius used for the borders of the component |
+| `--str-chat__tooltip-border-block-start`  | Top border of the component                             |
+| `--str-chat__tooltip-border-block-end`    | Bottom border of the component                          |
+| `--str-chat__tooltip-border-inline-start` | Left (right in RTL layout) border of the component      |
+| `--str-chat__tooltip-border-inline-start` | Right (lrft in RTL layout) border of the component      |
```
Comment (oliverlaz) [inline docs/theming/component-variables.mdx:580]:
```suggestion
| `--str-chat__tooltip-border-inline-end` | Right (left in RTL layout) border of the component      |
```

**Review by oliverlaz — APPROVED** [review]

File: docs/theming/component-variables.mdx (hunk 1 above)
Comment (szuperaz) [reply docs/theming/component-variables.mdx:548]: Will fix this after the ongoing PRs merged into theming-v2

**Pull request merged**